compiler.l

```diff
@@ -548,7 +548,7 @@
 (target js: (set (get global 'require) require))
 (target js: (define run eval))
 
-(target lua: (define load1 load))
+(target lua: (define load1 (or loadstring load)))
```

Comment:
Does it matter that this is `(or loadstring load)` and not the other way around?